sechub-sereco/src/main/java/com/daimler/sechub/sereco/importer/NetsparkerHtmlToAsciiDocConverter.java

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sereco.importer;
+
+import java.util.regex.Pattern;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.safety.Whitelist;
+import org.jsoup.select.Elements;
+
+/**
+ * Converts the Netsparker HTML to the AsciiDoc plain text format
+ */
+public class NetsparkerHtmlToAsciiDocConverter {
+    
+    private final static Pattern PATTERN = Pattern.compile("Â&nbsp;");   
```

Comment:
I believe, `NO_BREAK_SPACE_PATTERN` or something similar would be easier to understand.